src/ic3/enhancers/ingress/subscribeNewMessageAndThreadUpdate.ts

```diff
@@ -30,7 +30,12 @@ export default function createSubscribeNewMessageAndThreadUpdateEnhancer(): Adap
       getState(StateKey.Logger)?.logClientSdkTelemetryEvent(Microsoft.CRM.Omnichannel.IC3Client.Model.LogLevel.WARN,
         {
           Event: TelemetryEvents.UNKNOWN_MESSAGE_TYPE,
-          Description: `Adapter: Unknown message type; ignoring message ${message}`
+          Description: `Adapter: Unknown message type; ignoring message ${message?.messageid}`,
+          CustomProperties: stringifyHelper({
+            messageId: message?.messageid,
+            [StateKey.ChatId]: getState(StateKey.ChatId),
+            [StateKey.LiveworkItemId]: getState(StateKey.LiveworkItemId)
+          })
```

Comment:
same question about abstract this block in favor of reutilization

Reply:
same comment as above